modules/xmpp/JingleSessionPC.js

```diff
@@ -290,18 +324,32 @@ export default class JingleSessionPC extends JingleSession {
                 this,
                 this.peerconnection.iceConnectionState);
             switch (this.peerconnection.iceConnectionState) {
+            case 'checking':
+                this._iceCheckingStartedTimestamp = now;
```

Comment:
I haven't seen it going back to checking, but that's probably, because we do not try to send new candidates. I will check what happens with 'transport-replace' (JVB conference migration).